package-system/AWSGameLiftServerSDK/FindAWSGameLiftServerSDK.cmake.Linux

```diff
@@ -14,18 +14,27 @@ if (TARGET ${TARGET_WITH_NAMESPACE})
     return()
 endif()
 
+if ("${OPENSSL_VERSION}" STREQUAL "")
+    message(FATAL_ERROR "OpenSSL not detected. The OpenSSL dev package is required for O3DE")
+elseif ("${OPENSSL_VERSION}" VERSION_LESS "3.0.0")
+    set(SRC_OPENSSL_BASE openssl-1)
+else()
+    set(SRC_OPENSSL_BASE openssl-3)
+endif()
```

Comment:
Good idea to move these here instead of having multiple packages! I imagine we can do the same for `AWSNativeSDK` whenever we update that package?

Reply:
That is an issue that needs to be fixed :  https://github.com/o3de/o3de/issues/11640. Using this approach for the AWSNativeSDK will address that issue